code/src/components/app-range-slider/utils/parse-range-values/parseRangeValues.ts

```diff
@@ -0,0 +1,55 @@
+import checkWithinRange from "@/components/app-range-slider/utils/check-within-range/checkWithinRange";
+
+type ParseRangeValues = {
+  rangeStart: string;
+  rangeEnd: string;
+  min: number;
+  max: number;
+};
+
+const parseRangeValues = ({
+  rangeStart,
+  rangeEnd,
+  min,
+  max
+}: ParseRangeValues) => {
+  const numericRangeStart = parseInt(rangeStart);
+  const numericRangeEnd = parseInt(rangeEnd);
+
+  const isRangeStartValueValid = !isNaN(numericRangeStart);
+  const isRangeEndValueValid = !isNaN(numericRangeEnd);
+
+  const isRangeStartInputValid = Boolean(
+    checkWithinRange({
+      value: numericRangeStart,
+      min,
+      max
+    })
+  );
+  const isRangeEndInputValid = Boolean(
+    checkWithinRange({
+      value: numericRangeEnd,
+      min,
+      max
+    })
+  );
```

Comment:
`checkWithinRange` already returns boolean value
it's useless as a wrapper here `Boolean()`

Reply:
Really, i forgot to remove it after some refactoring. I did remove it